package gazeboPlugins.msgs;
import "vector3d.proto";

message GripperCommand
{
  enum COMMAND {
    MOVE = 0;
    GRAB = 1;
    RELEASE = 2;
  }
  required COMMAND cmd = 1;
  optional gazebo.msgs.Vector3d   direction = 2; 
}
